client/src/declarative/netflux/app.js

```diff
@@ -1,72 +1,5 @@
 import React from "react";
 
-export const MovieRow = ({movies = []}) => {
```

Comment:
I should probably leave a functional component example in to show that the application declaration doesn't have to be specified purely declaratively.